js/src/mpl_widget.js

```diff
@@ -363,17 +366,6 @@ var MPLCanvasView = widgets.DOMWidgetView.extend({
         }
     },
 
-    processPhosphorMessage: function(msg) {
-        MPLCanvasView.__super__.processPhosphorMessage.apply(this, arguments);
-
-        switch (msg.type) {
-        case 'resize':
-        // case 'after-show':
-            this.request_resize();
```

Comment:
Should we get rid of the `request_resize` method as well?

Reply:
I think that we want to keep the ability to request a resize. In the nbagg backend, the plot is resizable via a button, and that kind of functionality would need a call like this. Of course, what the request_resize function does needs to be changed quite a bit.

Reply:
See my latest commit where I rewrote all of this.